clipping_manager/models/clipping.py

```diff
@@ -79,6 +79,16 @@ def save(self, *args, **kwargs):
         self._generate_content_hash()
         super(Clipping, self).save(*args, **kwargs)
 
+    def clear(self):
```

Comment:
Actually, what you built here is basically the concept of soft deletion, right? We could also call it `soft_delete` as a result.

Reply:
You're right, it's a much better name 😀